src/amo/pages/AddonVersions/styles.scss

```diff
@@ -8,6 +8,26 @@
   @include respond-to(large) {
     display: flex;
   }
+
+  .CardList ul {
+    li {
+      margin-bottom: 0;
+    }
+
+    li.AddonVersions-warning {
```

Comment:
We should avoid the use of HTML elements in the SCSS files. So, ideally there should be no `li`, `ul` and `div` here.

Reply:
Instead, add/use CSS classes.

Reply:
The reason I used `ul` and `li` here is that I need to override [a style defined in another file](https://github.com/mozilla/addons-frontend/blob/2019.01.10/src/ui/components/CardList/styles.scss#L17), which already has `.CardList`, `ul` and `li` in it. I was having trouble overriding specificity here without including all of those, because the old rule kept overriding my new rule. Is there another trick to overriding specificity without needing all the elements?

Reply:
mmh, maybe that's because of `>` in the other style. So maybe that would work if you use `& > .AddonVersions-warning` instead of `li.AddonVersions-warning`.

Reply:
I was able to remove the `li` from this line, which is already in the latest version of the patch. It's the `ul` and `li` above it I was unable to get rid of because of specificity. It sounds like I can mark this as resolved now.

Reply:
yep, right. I commented before reviewing the latest revision of this patch. I should have left a comment to let you know that I was aware of the change.